node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1010,6 +1011,28 @@ pub(super) async fn run_chain_sync_task(
     Ok(most_recent_block_header)
 }
 
+async fn fetch_and_store_deploys(
```

Comment:
That could also be used in `sync_deploys_and_transfers_and_state` now.

Reply:
Certainly. I'll deduplicate the code in the PRs coming tomorrow.

Reply:
It's being prepared as a part of the bigger PR: https://github.com/rafal-ch/casper-node/commit/070f30d682d66f1581c042f8821919964cd5a946